### ISSUE TITLE
online demo 运行报错

### ISSUE THREAD

**Issue by haibins** [jerrywu001/vitepress-plugin-sandpack#3]
<img width="824" alt="image" src="https://github.com/jerrywu001/vitepress-plugin-sandpack/assets/29914762/b12f815b-fa9b-4921-aad6-722610cdd826">

```
  "devDependencies": {
    "vitepress": "1.0.0-beta.7",
    "vitepress-plugin-sandpack": "^0.3.0",
    "vue": "^3.3.4"
  },
```

请问一下是什么问题

**Comment by haibins** [comment]
试了一下 `"vitepress": "1.0.0-beta.3"` 版本，是可以的。online用的是最新版的，会有这个问题

**Comment by jerrywu001** [comment]
failed start from v1.0.0-beta.7.

https://github.com/vuejs/vitepress/issues/2720

**Comment by jerrywu001** [comment]
vitepress has fixed it, waiting for the pkg release.
https://github.com/vuejs/vitepress/commit/a56d608bec427ad51a9edb620d8fb01ebae29550

you can use the v1.0.0-beta.6 by now. @haibins

**Comment by haibins** [comment]
Thanks, another problem is that an error will be reported during the docs build,but it doesn't seem to affect the operation after packaging @jerrywu001 
<img width="1749" alt="image" src="https://github.com/jerrywu001/vitepress-plugin-sandpack/assets/29914762/3158b66e-a697-4dec-b0c7-61fee2e53686">

**Comment by jerrywu001** [comment]
这个没事，忽略即可，不影响dev & 打包部署

**Comment by haibins** [comment]
好的，具体是什么原因呀，有了解吗？看是不影响构建后的产物

**Comment by jerrywu001** [comment]
useData 在vitepress docs之外使用了，只是报warning，没事~

**Comment by haibins** [comment]
感谢，这里还遇到一个问题，在页面中的下载按钮没法下载
<img width="2003" alt="image" src="https://github.com/jerrywu001/vitepress-plugin-sandpack/assets/29914762/f6d6fb8d-1b32-4aff-a94d-9484d7e8d63b">


点开创建的codesandbox中是可以下载的，这个是什么原因呢
<img width="1927" alt="image" src="https://github.com/jerrywu001/vitepress-plugin-sandpack/assets/29914762/0651038d-8bc0-4ef3-9d75-af793c4ddd17">

**Comment by jerrywu001** [comment]
实际上底层渲染插件并没有指定可以修改iframe属性的参数 @haibins 

https://github.com/codesandbox/sandpack/blob/main/sandpack-client/src/types.ts#L4

**Comment by jerrywu001** [comment]
Fixed in v1.0.3

**problem-this-package-is-esm-only:** 

https://vitepress-sandbox.js-bridge.com/get-started/install.html#problem-this-package-is-esm-only


online demo:
https://stackblitz.com/edit/vitejs-vite-79ocfq?file=package.json
